pytorch-scatter.spec

```diff
@@ -9,14 +10,22 @@ Source: git+https://github.com/%{github_user}/pytorch_scatter.git?obj=%{branch}/
 
 BuildRequires: cmake
 Requires: pytorch
-%define build_flags -Wall -Wextra -pedantic %{?arch_build_flags} 
+%define build_flags -Wall -Wextra -pedantic %{?arch_build_flags}
+%define cuda_arch_float $(echo %{cuda_arch} | tr ' ' '\\n' | sed -E 's|([0-9])$|.\\1|' | tr '\\n' ' ')
 
 %prep
 %setup -n %{n}-%{realversion}
 # Make sure the default c++sdt stand is c++14
 grep -q 'CMAKE_CXX_STANDARD  *14' CMakeLists.txt
 sed -i -e 's|CMAKE_CXX_STANDARD  *14|CMAKE_CXX_STANDARD %{cms_cxx_standard}|' CMakeLists.txt
 
+USE_CUDA=OFF
+%if "%{cmsos}" != "slc7_aarch64"
+if [ "%{cuda_gcc_support}" = "true" ] ; then
+USE_CUDA=%{!?without_cuda:ON}
+fi
+%endif
```

Comment:
@iarspider , can you please update all these `pytorch-*.spec` ( where needed) and use something like
```
USE_CUDA=OFF
%if 0%{!?without_cuda:1}
if [ "%{cuda_gcc_support}" = "true" ] ; then
USE_CUDA=ON
fi
%endif
```